src/tools/rustfmt/src/attr.rs

```diff
@@ -17,6 +17,7 @@ use crate::shape::Shape;
 use crate::source_map::SpanUtils;
 use crate::types::{PathContext, rewrite_path};
 use crate::utils::{count_newlines, mk_sp};
+use rustc_ast::attr::AttributeExt;
```

Comment:
Apologies for the continued nit picks here, but the imports have again moved away from the other `rustc_ast` imports. Would you mind taking another look at the rustfmt changes.

Reply:
ugh, no the apologies are mine. I must have accidentally reverted that again at some point in my rebasing. Thanks for being thorough enough to notice cause I agree this shouldn't be like this...